controller.sh

```diff
@@ -10,8 +19,8 @@ if ! command -v docker &> /dev/null; then
 fi
 
 # Check for docker-compose
-if ! command -v docker compose &> /dev/null; then
```

Comment:
This looks pretty good, but go ahead and remove these checks here. There's no reason to check for the existence of docker compose twice!

Reply:
didnt see that sorry